pycytominer/tests/test_cyto_utils/test_cells.py

```diff
@@ -6,7 +6,7 @@
 from sqlalchemy import create_engine
 
 from pycytominer import aggregate, normalize
-from pycytominer.cyto_utils.cells import SingleCells
+from pycytominer.cyto_utils.cells import SingleCells, _sqlite_strata_conditions
```

Comment:
ah, I see, you're needing to input this as a new function. I typically think of `_single_leading_underscore` as being private methods, and [they're defined as "internal use" characters](https://pep8.org/#descriptive-naming-styles) in pep8. I'm wondering if I'm missing any critical insight that led you to separate out this method (This is kind of a duplicate comment of one previous)

Reply:
No real insight, no.  Yeah I just did it that way so I could easily import it to test it here.  And the leading underscore should denote to a user that it's private and never to be used directly for anything.